cfpb/consumerfinance.gov
# Fix iRegs wayfinder in appendices

The iRegs wayfinder isn't showing the right label for appendices. The label should be "Appendix [Letter]" when scrolling through appendix pages, but right now the wayfinder is showing the label of the paragraph in view. Looks like this was due to a change in what content gets passed to the page in `section`. See GHE/CFGOV/platform/issues/3957 for all the details.

---

## Changes

- Change the value of the `data-section` attribute from `{{section}}` to `{{section.title}}`

## How to test this PR

1. Visit any iRegs appendix, like /rules-policy/regulations/1005/A/
2. Confirm that the wayfinder label reads "Appendix {Letter}" the whole time when scrolling through the appendix page
3. Check to make sure the wayfinder remains correct when scrolling through sections and commentary and that nothing else about the wayfinder has changed

## Screenshots

![reg-e-appendix-expected](https://user-images.githubusercontent.com/1862695/101928441-94f93480-3ba3-11eb-8a9d-e97ef6375b26.png)

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
